app/controllers/provider_foreman_controller.rb

```diff
@@ -376,6 +381,14 @@ def cs_filter_tree_rec
     end
   end
 
+  def cm_job_templates_tree_rec
+    nodes = x_node.split('-')
+    type, _id = x_node.split("_").last.split("-")
```

Comment:
Couldn't this be `type = nodes.first.split("_").last`?  That should be 1 less call `String#split`